recipes/ninja/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set name = "NINJA_nj" %}
```

Comment:
IMO, Jinja2 variables should only be used when they simplify things, but in this case, "name" is used only once and then in its lowercase version. Why not leave out the "name" variable and set the name directly to "ninja_nj"?